webapp-django/crashstats/topcrashers/views.py

```diff
@@ -136,6 +138,16 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
+            # Number of startup crashes.
+            hit['startup_percent'] = 0
+
+            sig_startup = hit['facets']['histogram_uptime']
+            for row in sig_startup:
+                if row['term'] == 0:
+                    hit['startup_percent'] = (
```

Comment:
Yep, that's right. I just copied the way it was done in the old TCBS, but yours is better.